tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -2359,7 +2359,7 @@ describe('metrics reducers', () => {
         globalSettingsLoaded({
           partialSettings: {
             ignoreOutliers: true,
-            tooltipSortString: 'descending',
+            tooltipSortString: 'descending' as TooltipSort,
```

Comment:
Can you just use TooltipSort.DESCENDING?

Reply:
We can but, because the test is testing deserialization from browser storage, I prefer testing the string literal.

Reply:
The reasoning doesn't seem to apply here.

At this point in the code you are several layers removed from the serialized form. The only serialized form of the string exists in LocalStorage. The type globalSettingsLoaded says it will include a TooltipSort object, so that is the ideal type to use for the test.